src/frontend/src/features/layouts/components/mailbox-panel/components/mailbox-actions/_index.scss

```
@@ -5,45 +5,27 @@
-
-    &__refresh-icon,
-    &__refresh-spinner {
-        transition: opacity 150ms ease-out;
-    }
-
-    &__refresh-icon {
-        opacity: 1;
 
-        &--hidden {
-            opacity: 0;
+        @keyframes refreshIconSpin {
```

Stylelint (17.10.0)

[error] Expected keyframe name "refreshIconSpin" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)